core/ecschema-metadata/src/Metadata/CustomAttributeClass.ts

```diff
@@ -16,11 +16,12 @@ import { SchemaItem } from "./SchemaItem";
 
 /**
  * A Typescript class representation of an ECCustomAttributeClass.
- * @beta
+ * @public @preview
  */
 export class CustomAttributeClass extends ECClass {
   public override readonly schemaItemType = CustomAttributeClass.schemaItemType;
   public static override get schemaItemType() { return SchemaItemType.CustomAttributeClass; }
+  /** @internal */
   protected _appliesTo?: CustomAttributeContainerType;
```

Comment:
We could make this property private. In can be accessed through the mixinClass.appliesTo getter and is set through the setAppliesTo setter method. I haven't any of our code using the field directly outside this class.